pending_review/pull_request.go

```diff
@@ -176,27 +191,58 @@ func (s *PullRequestService) determineTypeOfChange(ctx context.Context, owner st
 		return nil, resp, fmt.Errorf("%w", ErrInvalidChange)
 	}
 
-	change, err := getDiff(files[0])
+	change, err := processChangedFiles(files)
 	if err != nil {
 		return nil, resp, err
 	}
 
+	return change, resp, nil
+}
+
+func processChangedFiles(files []*CommitFile) (*change, error) {
+	if len(files) < 1 {
+		return nil, fmt.Errorf("%w", ErrInvalidChange)
+	}
+
+	change, err := getDiff(files[0])
+	if err != nil {
+		return nil, err
+	}
+
+	addition := files[0].GetAdditions()
+	deletions := files[0].GetDeletions()
 	for _, file := range files[1:] {
 		obtained, err := getDiff(file)
 		if err != nil {
-			return nil, resp, err
+			return nil, err
 		}
 
-		if change.Recipe != obtained.Recipe { // PR should only be changing one recipe at a time
-			return nil, resp, fmt.Errorf("%w", ErrInvalidChange)
+		if change.Recipe != obtained.Recipe {
+			return nil, fmt.Errorf("%w", ErrInvalidChange)
 		}
 
 		if change.Change == NEW && obtained.Change == EDIT {
-			change.Change = EDIT // Any edit breaks the "new recipe" definition
+			change.Change = EDIT
 		}
+
+		addition += file.GetAdditions()
+		deletions += file.GetDeletions()
 	}
 
-	return change, resp, nil
+	//if len(files) <= 2 && addition <= 10 && deletions == 0 {
+	if len(files) <= 2 && (addition+deletions) <= 10 {
```

Comment:
I'd love some feedback 🙏

Reply:
The numbers seem ok for me. I worry that it will not be until we actually try these numbers that we'll know if they are balanced properly, but my gut says yes :)

Reply:
There a nice example, https://github.com/prince-chrismc/conan-center-index-pending-review/pull/109#issuecomment-1407459790

Reasonable is all I need on a first pass ;) I picked a few PRs while reviewing and said that seems about right 😆

Reply:
As first step I think is okay. We can update in case you feel different in the future.